packages/flutter_tools/test/integration.shard/isolated/native_assets_agp_version_test.dart

```diff
@@ -0,0 +1,123 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@Timeout(Duration(minutes: 10))
+library;
+
+import 'dart:io';
+
+import 'package:file/file.dart';
+import 'package:file_testing/file_testing.dart';
+
+import '../../src/common.dart';
+import '../test_utils.dart' show platform;
+import '../transition_test_utils.dart';
+import 'native_assets_test_utils.dart';
+
+const String packageName = 'package_with_native_assets';
+
+/// The AGP versions to run these tests against.
+final List<String> agpVersions = <String>[
+  '8.4'
+];
+
+/// The build modes to target for each flutter command that supports passing
+/// a build mode.
+///
+/// The flow of compiling kernel as well as bundling dylibs can differ based on
+/// build mode, so we should cover this.
+const List<String> buildModes = <String>[
+  'debug',
+  'profile',
+  'release',
+];
+
+void main() {
+  if (!platform.isMacOS && !platform.isLinux && !platform.isWindows) {
```

Comment:
@camsim99 @andrewkolos is this the right way to exclude platforms? I seem to recall a parameter that can be set on pure dart tests but am not sure if we have a documented way in integration tests?

Reply:
That is good enough for me. If we hear back from the tooling team then we can file a bug and fix it later.